package.json

```diff
@@ -4,22 +4,15 @@
   "private": true,
   "license": "AGPL-3.0",
   "scripts": {
-    "build": "yarn build-shared",
-    "cleanbuild": "yarn clean-all && yarn build-all",
-    "build-all": "CYPRESS_INSTALL_BINARY=0 yarn build-shared && yarn build-legacy && yarn build-ui && yarn build-root && yarn copy-build",
-    "build-root-css": "cd root && [ -f dist/assets/css/root-application.css ] || { yarn install; sass --load-path=../node_modules src/scss/base.scss dist/assets/css/root-application.css; }",
+    "build": "CYPRESS_INSTALL_BINARY=0 yarn --cwd ui install && yarn --cwd ui build && yarn copy-build",
     "build-shared": "cd shared && yarn install && yarn build",
-    "build-legacy": "cd legacy && yarn install && yarn build",
-    "build-root": "cd root && yarn install && yarn build",
-    "build-ui": "cd ui && yarn install && yarn build",
+    "build-all": "yarn build",
+    "cleanbuild": "yarn clean-all && yarn build",
     "cypress-run": "yarn cypress run",
     "cypress-run-a11y": "yarn cypress run --config integrationFolder=cypress/integration/accessibility",
     "cypress-open": "yarn cypress open",
     "percy": "./cypress/percy.sh",
-    "copy-build": "mkdir -p build && yarn copy-root && yarn copy-legacy-assets && yarn copy-ui-css",
-    "copy-legacy-assets": "cp -R legacy/dist/assets build/",
-    "copy-root": "cp -R root/dist/* build/",
-    "copy-ui-css": "cp ui/dist/static/css/* build/assets/css/",
+    "copy-build": "cp -r ui/build ./build",
```

Comment:
This step can be removed once we merge the ui workspace into the top level package.